crates/egui/src/containers/group.rs

```diff
@@ -0,0 +1,159 @@
+//! Frame container
+
+use crate::{layers::ShapeIdx, *};
+use epaint::*;
+
+/// A group of widgets with a unique id
+/// that can be used in centered layouts.
+#[derive(Clone, Copy, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+#[must_use = "You should call .show()"]
+pub struct Group {
```

Comment:
Is `Ui::group` going to be changed to use this? Or maybe renamed to avoid confusion?

Reply:
I don't know yet. This is all very much related to #4634 in that I would like a simple way to create a new `Ui` which has opt-in support for the new "grouping" behavior of this PR, as well as a frame (like `ui.group` does).